### ISSUE TITLE
empty container element on new collection page

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6809]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the container for the `Description` tab on the `New User Collection` page is empty and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/dashboard/collections/new?locale=en&collection_type_id=3 and run the Site Improve browser extension

![new_collection_container_element_empty](https://github.com/samvera/hyrax/assets/32644254/eefb2a0a-8d9c-4c73-acdb-d043797ee047)

**Comment by rjkati** [comment]
This issue is also present on the following pages:

- [Edit collection (all tabs)](https://dev.nurax.samvera.org/dashboard/collections/bc386j44x/edit?locale=en) - tablist
- [Add work by batch (relationships tab)](https://dev.nurax.samvera.org/batch_uploads/new?locale=en&payload_concern=GenericWork#relationships) - row below `Collection title` and `Action` headings
- [Add new work (relationships tab)](https://dev.nurax.samvera.org/concern/generic_works/new?locale=en#relationships) - row below `Collection title` and `Action` headings
- [Edit work (relationships tab)](https://dev.nurax.samvera.org/concern/generic_works/70795791z/edit?locale=en#relationships) - row below `Collection title` and `Action` headings
- [Edit file (all tabs)](https://dev.nurax.samvera.org/concern/file_sets/p2676v71k/edit?locale=en) - tablist

If it is preferable to create tickets for each occurrence, please comment here.

**Comment by christhepianist** [comment]
Confirmed that this is no longer an issue.
